stetho/src/main/java/com/facebook/stetho/inspector/domstorage/DOMStoragePeerManager.java

```diff
@@ -0,0 +1,86 @@
+package com.facebook.stetho.inspector.domstorage;
+
+import android.content.Context;
+import android.content.SharedPreferences;
+import com.facebook.stetho.inspector.helper.ChromePeerManager;
+import com.facebook.stetho.inspector.helper.PeerRegistrationListener;
+import com.facebook.stetho.inspector.helper.PeersRegisteredListener;
+import com.facebook.stetho.inspector.protocol.module.DOMStorage;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+
+public class DOMStoragePeerManager extends ChromePeerManager {
+  private final Context mContext;
+
+  public DOMStoragePeerManager(Context context) {
+    mContext = context;
+    setListener(mPeerListener);
+  }
+
+  private final PeerRegistrationListener mPeerListener = new PeersRegisteredListener() {
+    private final List<DevToolsSharedPreferencesListener> mPrefsListeners =
+        new ArrayList<DevToolsSharedPreferencesListener>();
+
+    @Override
+    protected synchronized void onFirstPeerRegistered() {
+      // TODO: We list the tags in Page.getResourceTree as well and those are the real fixed
```

Comment:
No, but a URL to github seems appropriate.  There is no task filed for this yet, and I actually don't like the idea of enforcing Facebook's internal policy of all TODOs are matched with a task.  I find that incredibly annoying, personally...